### PR TITLE
Feat/global styles

### PR DESCRIPTION
**Pull request by lee-cheung**
global styles for text, button, image, text input and scroll view

### CODE REVIEW

File: NativeApp/src/components/Login/Form/index.js [hunk 1]
```diff
@@ -56,23 +57,24 @@ class LoginForm extends Component {
...
           hasError && (
-            <Text validationText>
+            <Caption>
               Incorrect email or password
-            </Text>
+            </Caption>
           )
         }
-        <TouchableOpacity
+        <Button
```
Comment (kirsty-unosquare) [inline NativeApp/src/components/Login/Form/index.js:72]: Think we should use react-native-elements button instead for the buttons.

File: NativeApp/src/components/Common/Image/styles.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components';
+
+export const Image1 = styled.Image`
```
Comment (kirsty-unosquare) [inline NativeApp/src/components/Common/Image/styles.js]: Prefer if we just set the the image style within the components.

**Review by kirsty-unosquare — CHANGES_REQUESTED** [review]

File: NativeApp/src/components/Common/Caption/index.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+const Caption = (props) => {
+  const { children } = props;
+
+  return (
+    <ErrorText {...props}>{children}</ErrorText>
+  );
+};
+
+Caption.propTypes = {
+  children: PT.string.isRequired,
```
Comment (bgribz) [inline NativeApp/src/components/Common/Caption/index.js]: I think it's safer to use `PT.node` for any children, just on the off chance you need to pass a View containing multiple texts or something

File: NativeApp/src/components/Common/CardContainer/index.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+const CardContainer = (props) => {
+  const { children } = props;
+
+  return (
+    <View {...props}>{children}</View>
+  );
+};
+
+CardContainer.propTypes = {
+  children: PT.element.isRequired,
```
Comment (bgribz) [inline NativeApp/src/components/Common/CardContainer/index.js]: `PT.node`

File: NativeApp/src/components/Common/H1/index.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+const H1 = (props) => {
+  const { children } = props;
+
+  return (
+    <Text {...props}>{children}</Text>
+  );
+};
+
+H1.propTypes = {
+  children: PT.oneOfType([
```
Comment (bgribz) [inline NativeApp/src/components/Common/H1/index.js]: `PT.node`

File: NativeApp/src/components/Common/H2/index.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+const H2 = (props) => {
+  const { children } = props;
+
+  return (
+    <Text {...props}>{children}</Text>
+  );
+};
+
+H2.propTypes = {
+  children: PT.string.isRequired,
```
Comment (bgribz) [inline NativeApp/src/components/Common/H2/index.js]: `PT.node`

File: NativeApp/src/components/Common/H4/index.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+const H4 = (props) => {
+  const { children } = props;
+
+  return (
+    <Text {...props}>{children}</Text>
+  );
+};
+
+H4.propTypes = {
+  children: PT.string.isRequired,
```
Comment (bgribz) [inline NativeApp/src/components/Common/H4/index.js]: `PT.node`

File: NativeApp/src/components/Common/P/index.js [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+const P = (props) => {
+  const { children } = props;
+
+  return (
+    <Text {...props}>{children}</Text>
+  );
+};
+
+P.propTypes = {
+  children: PT.oneOfType([
```
Comment (bgribz) [inline NativeApp/src/components/Common/P/index.js]: `PT.node`

File: NativeApp/src/components/Common/ScrollView/index.js [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+const ScrollView = (props) => {
+  const { children } = props;
+
+  return (
+    <ScrollContainer {...props}>{children}</ScrollContainer>
+  );
+};
+
+ScrollView.propTypes = {
+  children: PT.oneOfType([
```
Comment (bgribz) [inline NativeApp/src/components/Common/ScrollView/index.js]: `PT.node`

**Review by bgribz — CHANGES_REQUESTED** [review]

File: NativeApp/src/components/Login/Form/index.js (hunk 1 above)
Comment (lee-cheung) [reply NativeApp/src/components/Login/Form/index.js:72]: made the changes :)

File: NativeApp/src/components/Common/Image/styles.js (hunk 2 above)
Comment (lee-cheung) [reply NativeApp/src/components/Common/Image/styles.js]: i have removed image file and styled within the component

File: NativeApp/src/components/Common/Caption/index.js (hunk 3 above)
Comment (lee-cheung) [reply NativeApp/src/components/Common/Caption/index.js]: cool, i have made the suggested changes :)

**Review by bgribz — APPROVED** [review]

**Review by kirsty-unosquare — APPROVED** [review]

**Pull request merged**